2015-12-09  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/unix/grantpt.c (grantpt): Call__basename instead of
	basename.

--- a/sysdeps/unix/grantpt.c
+++ b/sysdeps/unix/grantpt.c
@@ -195,7 +195,7 @@ grantpt (int fd)
       CLOSE_ALL_FDS ();
 # endif
 
-      execle (_PATH_PT_CHOWN, basename (_PATH_PT_CHOWN), NULL, NULL);
+      execle (_PATH_PT_CHOWN, __basename (_PATH_PT_CHOWN), NULL, NULL);
       _exit (FAIL_EXEC);
     }
   else
-- 
2.1.4

